<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>442</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>498</width>
    <height>442</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>498</width>
    <height>442</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>語学講座CS2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>481</width>
      <height>387</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0">
     <item row="10" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="Line" name="line">
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="0">
      <widget class="Line" name="line_2">
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_141">
       <item>
        <widget class="QCheckBox" name="checkBox_basic0">
         <property name="text">
          <string>小学生の基礎英語</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QCheckBox" name="checkBox_basic1">
         <property name="text">
          <string>中学生の基礎英語レベル１</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QCheckBox" name="checkBox_basic2">
         <property name="text">
          <string>中学生の基礎英語レベル２</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QCheckBox" name="checkBox_basic3">
         <property name="text">
          <string>中高生の基礎英語 in English</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QCheckBox" name="checkBox_timetrial">
         <property name="text">
          <string>英会話タイムトライアル</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QCheckBox" name="checkBox_kaiwa">
         <property name="text">
          <string>ラジオ英会話</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_19">
       <item>
        <widget class="QCheckBox" name="checkBox_business1">
         <property name="text">
          <string>ラジオビジネス英語</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_29">
       <item>
        <widget class="QCheckBox" name="checkBox_enjoy">
         <property name="text">
          <string>エンジョイ・シンプル・イングリッシュ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_21">
       <item>
        <widget class="QCheckBox" name="checkBox_vrradio">
         <property name="text">
          <string>ボキャブライダー</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <widget class="QCheckBox" name="checkBox_gendai">
         <property name="text">
          <string>ニュースで学ぶ「現代英語」</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox_chinese">
         <property name="text">
          <string>中国語</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="checkBox_hangeul">
         <property name="text">
          <string>ハングル講座</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="checkBox_french">
         <property name="text">
          <string>フランス語</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkBox_italian">
         <property name="text">
          <string>イタリア語</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QCheckBox" name="checkBox_german">
         <property name="text">
          <string>ドイツ語</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QCheckBox" name="checkBox_spanish">
         <property name="text">
          <string>スペイン語</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QCheckBox" name="checkBox_russian">
         <property name="text">
          <string>ロシア語</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QCheckBox" name="checkBox_portuguese">
         <property name="text">
          <string>ポルトガル語</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QCheckBox" name="checkBox_arabic">
         <property name="text">
          <string>アラビア語</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_111">
       <item>
        <widget class="QCheckBox" name="checkBox_japanese">
         <property name="text">
          <string>日本語</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_26">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>　　　　　拡張子：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_extension">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>aac</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>m4a</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mp3-128k-S</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mp3-64k-S</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mp3-48k-S</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mp3-40k-M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mp3-32k-M</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mp3-24k-M</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_25">
       <item>
        <widget class="QPushButton" name="downloadButton">
         <property name="text">
          <string>レコーディング</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_28">
       <item>
        <widget class="QCheckBox" name="checkBox_next_week2">
         <property name="text">
          <string>[前週]放送翌週月曜(10時)から１週間</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_22">
       <item>
        <widget class="QCheckBox" name="checkBox_skip">
         <property name="text">
          <string>既存のファイルはスキップ</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBox_this_week">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>257</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>公開中のストリーミング</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_next_week">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>712</y>
      <width>257</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>次週のストリーミング</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>491</x>
      <y>742</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>日分</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="past_days">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>436</x>
      <y>739</y>
      <width>45</width>
      <height>22</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_past_week">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>740</y>
      <width>155</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>過去のストリーミング</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_13">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>247</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>リトル・チャロ2</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>268</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_14">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ニュースで英会話　</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_enews">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>音声と動画</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>音声のみ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>動画のみ</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>268</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QCheckBox" name="checkBox_15">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ニュースで英会話（読み直し音声）</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>610</y>
      <width>270</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QCheckBox" name="checkBox_shower">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ABCニュースシャワー</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_shower">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>音声と動画</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>音声のみ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>動画のみ</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>690</y>
      <width>162</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>拡張子(ABCニュース以外)：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>650</y>
      <width>269</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_24">
     <item>
      <widget class="QCheckBox" name="checkBox_enews_past">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>「ニュースで英会話」過去分</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>750</y>
      <width>209</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_23">
     <item>
      <widget class="QCheckBox" name="checkBox_keep_on_error">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ダウンロード未完でも変換</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>checkBox_basic0</tabstop>
  <tabstop>checkBox_basic1</tabstop>
  <tabstop>checkBox_basic2</tabstop>
  <tabstop>checkBox_basic3</tabstop>
  <tabstop>checkBox_timetrial</tabstop>
  <tabstop>checkBox_kaiwa</tabstop>
  <tabstop>checkBox_business1</tabstop>
  <tabstop>checkBox_enjoy</tabstop>
  <tabstop>checkBox_vrradio</tabstop>
  <tabstop>checkBox_gendai</tabstop>
  <tabstop>checkBox_chinese</tabstop>
  <tabstop>checkBox_hangeul</tabstop>
  <tabstop>checkBox_french</tabstop>
  <tabstop>checkBox_italian</tabstop>
  <tabstop>checkBox_german</tabstop>
  <tabstop>checkBox_spanish</tabstop>
  <tabstop>checkBox_russian</tabstop>
  <tabstop>checkBox_portuguese</tabstop>
  <tabstop>checkBox_arabic</tabstop>
  <tabstop>checkBox_japanese</tabstop>
  <tabstop>checkBox_skip</tabstop>
  <tabstop>comboBox_extension</tabstop>
  <tabstop>downloadButton</tabstop>
  <tabstop>past_days</tabstop>
  <tabstop>comboBox_enews</tabstop>
  <tabstop>checkBox_next_week</tabstop>
  <tabstop>checkBox_15</tabstop>
  <tabstop>checkBox_14</tabstop>
  <tabstop>checkBox_this_week</tabstop>
  <tabstop>checkBox_13</tabstop>
  <tabstop>checkBox_shower</tabstop>
  <tabstop>comboBox_shower</tabstop>
  <tabstop>checkBox_enews_past</tabstop>
  <tabstop>checkBox_keep_on_error</tabstop>
  <tabstop>checkBox_past_week</tabstop>
  <tabstop>checkBox_next_week2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>downloadButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>download()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>download()</slot>
 </slots>
</ui>
